superset/models/dashboard.py

```diff
@@ -311,6 +311,10 @@ def alter_positions(
         # copy slices object as Slice.import_slice will mutate the slice
         # and will remove the existing dashboard - slice association
         slices = copy(dashboard_to_import.slices)
+
+        # Clearing the slug to avoid conflicts
+        dashboard_to_import.slug = None
```

Comment:
This was raising when loading the same dashboard (with a slug) twice. Currently the authority on whether to insert or update is the `remote_id` and didn't want to tangle the slug into that. Imported dashboards loose their slugs. We could lookup whether there will be a conflict or not and keep the slug / suffix it, but for now I'm just fixing this bug.

Reply:
Removing the slug might be unexpected behavior for the end user. I wonder if we should raise an alert toast if the slug has been removed until we figure out a more permanent solution?

Reply:
So let's say I raise `Slug already exists!` user should do what? Go to change the slug from the source and re-export? I feel like we'd need to wizard that up and be like "Want to overwrite or create a new one?". Currently it'd be hard to do that.

I guess the form could offer "overwrite based on slug" checkbox or similar... ?

Reply:
TBH this is one of those things where it'd be really valuable to get usage statistics on how much slugs are used, how often slugged dashboards are exported + the slug is expected to work upon import. My guess is not so often, but who knows. But for the time being I'm comfortable with clearing the slug to avoid getting errors on repeated imports (idempotency FTW).